.travis.yml

```diff
@@ -45,7 +46,10 @@ before_script:
 
 script:
     - make
-    - if [ -z "$CROSS_COMPILE" ]; then make test; fi
+    - if [ -n "$CROSS_COMPILE" ]; then
+          export EXE_PRE="wine " EXE_EXT=".exe" WINEPREFIX=`pwd`;
```

Comment:
Normally it's possible to simply start Windows .exe files as if they are plain Linux binaries. Of course it depends on /proc/sys/fs/binfmt_misc being mounted and wine being registered as handler for PE-COFF binaries. But these are available immediately after installation of the 'wine' package... I suppose question is why it didn't work here, or in other words if we can achieve the goal without EXE_PRE.